src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/config/GivensBuilder.java

```diff
@@ -215,6 +219,60 @@ public GivensBuilder iUpdateVariablesOfTaskById(String taskId, Map<String, Camun
         return processVariables.getProcessVariablesMap();
     }
 
+    public Map<String, CamundaValue<?>> createDefaultTaskVariablesWithWarnings(String caseId) {
+        String values = "[{\"warningCode\":\"Code1\", \"warningText\":\"Text1\"}, "
+            + "{\"warningCode\":\"Code2\", \"warningText\":\"Text2\"}]";
+
+        CamundaProcessVariables processVariables = processVariables()
+            .withProcessVariable("caseId", caseId)
+            .withProcessVariable("jurisdiction", "IA")
+            .withProcessVariable("caseTypeId", "Asylum")
+            .withProcessVariable("region", "1")
+            .withProcessVariable("location", "765324")
+            .withProcessVariable("locationName", "Taylor House")
+            .withProcessVariable("staffLocation", "Taylor House")
+            .withProcessVariable("securityClassification", "PUBLIC")
+            .withProcessVariable("group", "TCW")
+            .withProcessVariable("name", "task name")
+            .withProcessVariable("taskId", "reviewTheAppeal")
+            .withProcessVariable("taskType", "reviewTheAppeal")
+            .withProcessVariable("taskCategory", "Case Progression")
+            .withProcessVariable("taskState", "unconfigured")
+            .withProcessVariable("dueDate", now().plusDays(2).format(CAMUNDA_DATA_TIME_FORMATTER))
```

Comment:
```suggestion
            .withProcessVariable("dueDate", now().plusDays(10).format(CAMUNDA_DATA_TIME_FORMATTER))
```

Explanation by default when no sorting is provided we will sort task by due date so your task must be in the far future to be on the top 50 tasks, however, we have too many tests running and setting the trade 2 days in the future at 4pm which makes it a bit harder to find the solution for this is to make it so it's somewhere like 10 days in the future that way you will not be competing against task config etc..